geowebcache/core/src/main/resources/org/geowebcache/rest/webresources/openlayers3/ol.js

```diff
@@ -0,0 +1,998 @@
+// OpenLayers 3. See http://openlayers.org/
```

Comment:
This is marked as a new file, and I can't find a corresponding deleted file.

Elsewhere, I see 
```
...che/web/src/main/webapp/openlayers3/ol.js → ...s/org/geowebcache/rest/webresources/ol.js
```

So it looks like there might be a duplicate ol3 file?

Reply:
Ah I made a duplicate when I ran into trouble with subdirectories and forgot to delete it, this is the correct one, it's the one that did get marked as a rename that shouldn't be there.  Thanks for catching that.